### ISSUE TITLE
About Caffe's Scale layer's weights and ACL's BatchNorm

### ISSUE THREAD

**Issue by gunjupark** [ARM-software/ComputeLibrary#775]
![image](https://user-images.githubusercontent.com/36186765/68642551-e6269700-0551-11ea-81e4-b2ce81572b27.png)

**Platform: Odroid N2 - Mali G52**

**Operating System: Ubuntu mate 18.04**

I want to use ACL's CLBatchNormalizationLayer in Caffe based framework.

But caffe's BatchNormLayer is separated into 3 parts 
( BatchNorm : normalize[mean, var] , Scale : scale[gamma] , shift[beta] )

In Caffe's pretrained model, caffe batchnorm layers's weights(mean, var) can be directly applied to 
ACL's CLBatchNormalizationLayer. --> output has no problem

But when I put Caffe Scale Layer's gamma, beta into ACL BatchNorm, output is weird value.

How ACL can make graph with weights from caffemodel? (please focus scale layer's weights)

Thanks you

**Comment by gunjupark** [comment]
Sorry 
There was a mistake in the code modification process.
Caffe Scale Layer's weights (gamma, beta) is same in ACL batchnorm layer's
